backend/compact-connect/compact-config/aslp/kentucky.yml

```diff
@@ -0,0 +1,13 @@
+# FOR TESTING PURPOSES ONLY, NOT ACTUAL CONFIGURATION
+stateName: "kentucky"
```

Comment:
Because we're referencing states by postal abbreviation (`oh`, `ne`, `ky`) everywhere in the API, I bet we'll need to include those in here somewhere to connect the dots.